tests/test_definitions/column_aggregate_expectations/expect_column_max_to_be_between.json

```diff
@@ -150,7 +150,7 @@
       },
       "out": {
         "success": true,
-        "observed_value": "2001-01-01 12:00:01"
+        "observed_value": "2001-01-01T12:00:01"
```

Comment:
Is this problematic? I think the datetime parsing logic I added does this - we were previously erroneously comparing strings so I'm leaning towards this being more appropriate

Reply:
Yeah, agreed that this seems better.